src/extensions/IDictionaryExtensions.cs

```diff
@@ -11,7 +11,7 @@ namespace Microsoft.Kiota.Abstractions.Extensions
     /// <summary>
     /// Extension methods for the <see cref="IDictionary"/>
     /// </summary>
-    public static class IDictionaryExtensions
+    internal static class IDictionaryExtensions
```

Comment:
I believe this may break the depending packages as some of the other kiota packages do reference the `TryAdd` method after the downgrade to netstandard2.0

Reply:
```suggestion
    public static class IDictionaryExtensions
```